docs/source/models/cause.rst

```diff
@@ -275,14 +273,155 @@ Birth Prevalence
+in a specified timeframe and population. The count value of new cases of the 
+condition of interest will always be the numerator of incidence measures. The 
+denominator of incidence measures is somewhat more complex and is critical to 
+ensuring an accurate data source to inform cause model transition rates. 
+
+Two incidence measures relevant to cause model transition rate data sources 
+using GBD results are discussed in this section, including measures we refer 
+to as **incidence in the total population** (as estimated by the GBD study) 
+and **incidence in the susceptible** (or *at-risk*) **population.** These 
+measures are defined using the following key concepts:
+
+  **Person-time:** person-time is a measure of the number of individuals 
+  multiplied by the amount of time they individually occupy the population 
+  of interest. Notably, the population of interest varies depending on context 
+  and can be defined by age group, sex, location, time, disease status, etc.
+
+    For example, if one individual is occupies the population of interest for 
+    two years, they contribute two person-years. If another individual is in 
+    our population of interest for 6 months, they contribute 0.5 person-years.
+    Together, these two individuals contribute a total of 2.5 person-years.
+
+  **Susceptible or At-Risk Population:** the susceptible population, also 
+  referred to as the at-risk population, is defined as the population that *
+  does not* have the condition of interest; in other words, the susceptible 
+  population that is at risk of developing the condition. Notably, the number 
+  of individuals in this population will change overtime as the following 
+  events occur:
+
+     - Members of the at-risk population develop the condition and are no longer susceptible
+     - Members of the at-risk population die and are no longer susceptible 
+     - Individuals are born or age into the at-risk population and become susceptible
+     - Individuals age out of the at-risk population and are no longer susceptible
+     - Individuals with the condition recover from the condition and re-enter the at-risk population as susceptible (in the case of conditions with remission)
+
+**Total Population Incidence Rate** is estimated by the Global Burden of 
+Disease Study by estimating the number of incident cases that occur in one 
+year and scaling this value per 100,000 individuals of a specified population.
+
+.. math::
+
+  \frac{n_\text{incident cases}}{\text{person-time}_\text{total population}}
+
+Because the denominator of this measure is not specific to a particular cause 
+model state, it is **not** an appropriate data source for cause model 
+transition rates between states. 
+
+.. note:: 
+
+  GBD estimates of total population incidence rate require transformation 
+  prior to use as a cause model transition probability data source (see below 
+  for more detail).
```

Comment:
This sounds a little too absolute to me. Do we **always** transform the data prior to use, or are there cases (e.g. if we don't have good prevalence data but we know the prevalence is small), where we would just use the total population incidence rate in the model as a close-enough approximation? @collijk , any input on this?

Also, saying that the GBD estimates of total population incidence rates are "not an appropriate data source" makes it sound like we can't use them and we'll have to get data somewhere else. But that's not true - we do use them, we just have to combine this estimate with a different estimate to get the number we want.  So perhaps changing the wording of that sentence would help.

Reply:
For all the models we use now, we always perform the transformation.  That is up to y'all though.

Reply:
I will add some clarity to the language about how incidence as retrieved from GBD final results is not an appropriate data source **before** transformation to the at-risk population. 

Also, for context: as discussed with @collijk, our transformation is simply un-doing a transformation that GBD made to get incidence in the total population from an estimate of incidence in the susceptible population.